test/1_date_time/test.js

```diff
@@ -264,11 +281,72 @@ define(['BlinkForms', 'testUtils'], function (Forms, testUtils) {
           assert($input.val(), "no value assigned to $input");
         });
       });
-
       defineModelToViewTests();
       defineViewToModelTests();
     });
 
+    suite('picker value assignment', function () {
+      var pickadate = [
+        'datenonative',
+        'datetimenonative',
+        'datenow',
+        'datefromnowplus',
+        'datefromdate',
+        'datetimenow',
+        'dateTimeHiddenNone',
+        'dateTimeHiddenNow',
+        'dateTimeHiddenNowP',
+        'dateTimeHddenNowPM',
+        'dateTimePickerNone',
+        'dateTimePickerNow',
+        'dateTimePickerNowP',
+        'dateTimePckerNowPM'
+      ];
+
+      test('element value assignment reflected in picker, model, view ', function (done) {
+        setTimeout(function () {
```

Comment:
ok no worries :)

i've noticed that sometimes things fail due to timing. its pretty painful when it seems random :(